dev-libs/girara/girara-9999.ebuild

```diff
@@ -1,67 +1,48 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
 
-inherit multilib toolchain-funcs virtualx
-[[ ${PV} == 9999* ]] && inherit git-2
+inherit meson virtualx
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+
```

Comment:
Drop the blank line and reorder inherit last.

Reply:
Also put some stars around 9999, for this and the other ebuilds. Helpful if say
`dev-libs/girara/girara-1.0.9999.ebuild` (eg, live ebuild for the 1.0 branch) becomes
a thing.

Reply:
Hum, I see your point, I thought about a single 9999 version, thanks.